nemo_skills/evaluation/metrics/base.py

```
@@ -71,44 +71,37 @@ def _add_std_metrics(self, metrics_dict):
                 std_dev_across_runs = np.std(run_averages, ddof=1)
                 std_err_across_runs = std_dev_across_runs / math.sqrt(k)
 
+                # Average of metric values across runs
+                avg = np.mean(run_averages)
+
                 # Average of per-sample standard deviations
-                sample_std_devs = []
-                for scores in scores_list:
-                    sample_std_devs.append(np.std(scores[:k], ddof=1))
-                avg_sample_std_dev = sum(sample_std_devs) / len(sample_std_devs)
+                sample_std_devs = [np.std(scores, ddof=1) for scores in zip(*scores_list[:k])]
```

Ruff (0.14.0)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)